binstub

```diff
@@ -58,7 +58,12 @@ while IFS= read -r line; do
 
       case "$argument" in
         "$pattern" ) ;;
-        * ) result=1 ;;
+        * )
```

Comment:
This looks similar to my PR #20. Can you test that and if that suits you remove this from this PR